app/views/state_file/questions/return_status/_pending.html.erb

```diff
@@ -9,15 +9,31 @@
 </h1>
 
 <p>
-<% if current_intake.email_notification_opt_in_yes? && current_intake.sms_notification_opt_in_yes? %>
-  <%= t('.receive_email_sms') %>
-<% elsif current_intake.email_notification_opt_in_yes? %>
-  <%= t('.receive_email') %>
-<% elsif current_intake.sms_notification_opt_in_yes? %>
-  <%= t('.receive_sms') %>
-<% end %>
+  <%= t(".waiting_for acceptance") %>
 </p>
 
 <p>
-  <%= app_time.before?(Rails.configuration.state_file_end_of_in_progress_intakes) ? t('.more_than_two_days') : t('.more_than_two_days_closed_html') %>
-</p>
\ No newline at end of file
+  <% if current_intake.email_notification_opt_in_yes? && current_intake.sms_notification_opt_in_yes? %>
+    <%= t('state_file.questions.submission_confirmation.edit.email_text_update') %>
+  <% elsif current_intake.email_notification_opt_in_yes? %>
+    <%= t('state_file.questions.submission_confirmation.edit.email_update') %>
+  <% elsif current_intake.sms_notification_opt_in_yes? %>
+    <%= t('state_file.questions.submission_confirmation.edit.text_update') %>
+  <% end %>
+</p>
+
+<p>
+  <%= t('.check_spam') %>
+</p>
```

Comment:
does this `.check_spam` message make sense if they haven't opted in to any communications?

Reply:
confirmed with Anu/Madison that this should only show if they signed up with email
https://codeforamerica.atlassian.net/browse/FYST-1340?focusedCommentId=13932